closed/OpenJ9.gmk

```diff
@@ -164,13 +172,43 @@ endef
 # ----------------------
 # param 1 = The jdk/jre directory name
 # param 2 = The jdk/jre directory to add openj9 content
+ifeq (xr32,$(OPENJ9_PLATFORM_CODE))
```

Comment:
What are the files that are missed if the 'buildjvm' target(s) aren't called?

Reply:
Possibly not a complete list, but these are the libraries that for a while I was manually copying in order to compose the build:
```
libomrsig.so libj9vm29.so libj9thr29.so libj9hookable29.so libj9prt29.so \
  libffi29.so libhyprtshim29.so libhythr.so libj9dmp29.so libj9trc29.so \
  libj9zlib29.so libj9jit29.so libj9gc29.so libj9vrb29.so libj9jvmti29.so \
  libjclse9_29.so libj9shr29.so libjvm.so ../j9vm/libjvm.so
```
There seemed to be good overlap between this set and the files defined for OPENJ9_JAVA_BASE_LIBRARIES and OPENJ9_SHARED_CLASSES_LIBRARIES.